src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -238,82 +237,79 @@ const LiveVehicle = ({
   );
 
   const circleColor = useTransportationColor(mode, subMode);
-  const circleStyle = ((): CircleLayerStyleProps => {
-    if (isError)
-      return {
-        circleColor:
-          theme.interactive.interactive_destructive.disabled.background,
-        circleRadius: 20,
-        circleStrokeColor:
-          theme.interactive.interactive_destructive.default.background,
-        circleStrokeWidth: 2,
-      };
-    if (isLoading || isStale)
-      return {
-        circleColor: theme.interactive.interactive_1.disabled.background,
-        circleRadius: 20,
-        circleStrokeColor: theme.interactive.interactive_1.default.background,
-        circleStrokeWidth: 2,
-      };
-    return {
-      circleColor,
-      circleRadius: 22,
-      circleStrokeWidth: 0,
-    };
-  })();
+
+  let circleBackgroundColor = circleColor;
+  let circleBorderColor = 'transparent';
+  if (isError) {
+    circleBackgroundColor =
+      theme.interactive.interactive_destructive.disabled.background;
+    circleBorderColor =
+      theme.interactive.interactive_destructive.default.background;
+  }
+  if (isLoading || isStale) {
+    circleBackgroundColor = theme.interactive.interactive_1.disabled.background;
+    circleBorderColor = theme.interactive.interactive_1.default.background;
+  }
 
   if (!vehicle.location || zoomLevel < FOLLOW_MIN_ZOOM_LEVEL) return null;
 
+  const iconCircleSize = 44;
```

Comment:
It comes from (circleRadius: 20 + circleStrokeWidth: 2)*2 that was used before this commit. It could be that it should be defined in the design system, I'm not sure.